it/uk/gov/hmrc/agentassurance/controllers/AgentAssuranceControllerISpec.scala

```diff
@@ -458,6 +460,77 @@ class AgentAssuranceControllerISpec extends IntegrationSpec
     }
   }
 
+  feature("/overseas-agents/amls") {
+    val amlsCreateUrl = s"http://localhost:$port/agent-assurance/overseas-agents/amls"
+
+    val arn = Arn("AARN0000002")
+
+    val amlsDetails = AmlsDetails("supervisory", "0123456789", LocalDate.now())
```

Comment:
should localDate be part of amls details?  I think it should be part of OverseasAmlsEntity class rather than AmlsDetails class